images/virtualization-artifact/.golangci.yaml

```
@@ -21,7 +21,7 @@ linters-settings:
     sections:
       - standard
       - default
-      - prefix(github.com/deckhouse/)
+      - prefix(github.com/deckhouse/virtualization/)
```

```suggestion
      - prefix(github.com/deckhouse/)
```
Sorry, I was wrong. We still use "virtualization-controller" prefix.